scripts/e2e-saucelabs.sh

```diff
@@ -6,11 +6,13 @@ source $OKTA_HOME/$REPO/scripts/setup.sh
 setup_service java 1.8.222
 setup_service google-chrome-stable 89.0.4389.72-1
 
-export RUN_SAUCE_TESTS=true
+export RUN_SAUCE_TESTS=ie11,edge
```

Comment:
`RUN_SAUCE_TESTS` can be list of `ie11`, `edge`, `mobile` separated with `,`

`mobile` is not included here (and in`e2e-saucelabs-v3` as well) because it's flaky (iPad can show "Let's browse!" message for localhost instead of rendering test app), and there is a separate Bacon test named `e2e-saucelabs-mobile` that runs tests specially for mobile platforms

Question: do we need `edge`? Since MS Edge is based on Chromium

Reply:
we can leave it `edge` on the list

Reply:
Do you mean remove `ie11` and leave only `edge`?
```
export RUN_SAUCE_TESTS=edge
```
But then we lose ability to easily detect IE11 regressions